osmapi/OsmApi.py

```diff
@@ -1963,33 +1973,33 @@ def _http(self, cmd, path, auth, send):  # noqa
                 self._conn = self._get_http_connection()
```

Comment:
I think it's a good idea to use a session object of requests to potentially re-use the TCP connection for following requests. I would call it `_session` and probabably rename `_get_http_connection()` to `_get_http_session()`. WDYT?